src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,12 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine[] machines = {new Truck(), new Bulldozer(), new Excavator()};
 
+        for (int i = 0; i < machines.length; i++) {
+            machines[i].doWork();
+            machines[i].stopWork();
+        }
```

Comment:
This is better to use `for-each`, if you don't manipulate with indexes, but just do linear read over array.